templates/pipeline_run_pruner.sh

```diff
@@ -0,0 +1,22 @@
+set -exuo pipefail
+
+delete_threshold=$(date --date="{{ osbs_pruner_pipeline_runs_minutes_old }} minutes ago" --iso-8601=seconds --utc)
+OC='oc -n {{ osbs_ocp_namespace }}'
+
+old_pipelineruns=$(
+    $OC get pipelinerun -o json | \
+    jq '.items[] | select(.status.completionTime != null) | select(.status.completionTime < "'${delete_threshold}'") | .metadata.name' -r | \
```

Comment:
Comparing timestamps (https://stedolan.github.io/jq/manual/#Dates) might be more robust, but this should work too

Reply:
we are using it like this all the time and it works

Reply:
Ack, works for me